### PR TITLE
iss augmented function - a series of call backs

### PR DESCRIPTION
**Pull request by hennigk**

### CODE REVIEW

File: iss.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        //a function to add the property toRadians()
+    Number.prototype.toRadians = function() {
+        return this * Math.PI / 180;
+    };
+    
+    //a self-invoking function to get the coordinates of the ISS 
+    //and calls haversince calculations
+    //function CALL FOUR
+    (function getISSCords () {
+    var request = require('request');
```
Comment (nyanofthemoon) [inline iss.js]: It is general good practice for modules to be included at the beginning of the file.

File: iss.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  });
+ } 
+
+/*a function that gets the lat and long of the user's inputed city.
+calls a function to calculate the distance between the users city
+and the iss using the parameters from the request*/
+//function CALL TWO
+function callMaps(city) {
+    var mapsHttps = "https://maps.googleapis.com/maps/api/geocode/json?address="+city;
+    var request = require('request');
```
Comment (nyanofthemoon) [inline iss.js]: It is general good practice for modules to be included at the beginning of the file.

File: iss.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
+
+//a program that displays the location of the space station!
+
+//prompt to user ask for their location. it then calls the map function.
+//function CALL ONE
+function getLocation() {
+var prompt = require('prompt');
```
Comment (nyanofthemoon) [inline iss.js]: It is general good practice for modules to be included at the beginning of the file.

Comment (hennigk) [inline iss.js]: Ok, that makes sense! 
I made the changes at this commit: 
iss-augmented 3d2817e

**Pull request closed without merging**